vllm/_custom_ops.py

```diff
@@ -532,6 +532,58 @@ def cutlass_scaled_mm_azp(a: torch.Tensor,
     return out
 
 
+def cutlass_scaled_sparse_mm_supports_fp8(cuda_device_capability: int) -> bool:
+    return torch.ops._C.cutlass_scaled_sparse_mm_supports_fp8(
+        cuda_device_capability)
+
+
+def cutlass_compress_entry(a: torch.Tensor) \
+    -> Tuple[torch.Tensor, torch.Tensor]:
+    assert (a.dtype is torch.int8 or a.dtype is torch.float8_e4m3fn or \
+            a.dtype is torch.bfloat16 or a.dtype is torch.float16)
+
+    # Not exactly sure what the right value would be based on cutlass definitions
+    # Let's assume e.dtype: torch.uint8 so elemsPerElemE = 8b / 2b_per_nz = 4
+    elemsPerElemE = 4
```

Comment:
Can this issue be resolved before landing?

Reply:
Done